README.md

```diff
@@ -969,6 +969,13 @@ The information required to connect to Kafka will be automatically set for you b
 
 Before you begin fixing issues or adding new features to the Kogito Operator, see [Contributing to the Kogito Operator](docs/CONTRIBUTING.MD) and [Kogito Operator architecture](docs/ARCHITECTURE.MD).
 
+### Requirements
```

Comment:
Please add this section to Table of Contents

Reply:
Maybe would be better to rename to Build requirements, as there is already `Kogito Operator requirements` section too.

Reply:
Done